app/forms/reset_password_form.rb

```diff
@@ -6,13 +6,15 @@ class ResetPasswordForm
 
   validate :valid_token
 
-  def initialize(user)
+  def initialize(user, options = {})
```

Comment:
Could we list the keyword argument and its default in the arguments instead, which may make it clearer what the method is expecting to receive?

```suggestion
  def initialize(user, validate_confirmation: false)
```